app/helpers/summary_helper.rb

```diff
@@ -44,13 +44,15 @@ def build_data_row(object, field)
   end
 
   def value_style(value)
-    ['column-two-thirds',
-     (
-      {
-        '✓' => ' summary-result passed',
-        '✗' => ' summary-result failed',
-        'W' => ' summary-result part'
-      }[value.to_s.first] || '')
-    ].join
+    styles = ['column-two-thirds']
+    case value
+    when /^✓/
+      styles << 'summary-result passed'
+    when /^x/
+      styles << 'summary-result failed'
+    when /^Waiting for/
+      styles << 'summary-result part'
+    end
+    styles
```

Comment:
This is perfectly fine, I prefer a `tap` solution though. When the return of the method is only the variable, which is constructed in the method, I think it's cleaner, but I'll leave it completely up to you.

```ruby
['column-two-thirds'].tap do |styles|
  case value
    when /^✓/
      styles << 'summary-result passed'
    when /^x/
      styles << 'summary-result failed'
    when /^Waiting for/
      styles << 'summary-result part'
  end
end
```

If you prefer the current solution, merge in! 👍